internal/core/common_config/main.go

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright 2023 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package common_config
+
+import (
+	"context"
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/flags"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router) {
```

Comment:
Remove router parameter
```suggestion
func Main(ctx context.Context, cancel context.CancelFunc) {
```